sssd_test_framework/utils/authentication.py

```diff
@@ -413,23 +412,29 @@ def __init__(self, host: MultihostHost) -> None:
         self.opts = "-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no"
         """SSH CLI options."""
 
-    def password(self, username: str, password: str) -> bool:
+    def password(self, username: str, password: str, output: bool = False) -> bool | str:
         """
         SSH to the remote host and authenticate the user with password.
 
-        :param name: User name.
-        :type name: str
+        :param username: Username.
+        :type username: str
         :param password: User password.
         :type password: str
-        :return: True if authentication was successful, False otherwise.
-        :rtype: bool
+        :param output: Log output.
+        :type output: bool
+        :return: True if authentication was successful, False otherwise. If output is true, return output.
+        :rtype: bool | str
         """
+        log = "0"
+        if output is True:
+            log = "1"
```

Comment:
I don't understand why you need to convert the boolean to a numeric value. Can you explain?

Reply:
I think its nicer to use True/False instead if "1" | "0".